packages/credentials/package.json

```diff
@@ -75,9 +75,9 @@
   },
   "dependencies": {
     "@sphereon/pex": "2.1.0",
-    "@web5/common": "0.2.2",
-    "@web5/crypto": "0.2.4",
-    "@web5/dids": "0.2.4"
+    "@web5/common": "0.2.3",
```

Comment:
These are the latest versions right?

Reply:
Yes -- `0.4.0` for both dids and crypto -- `0.2.3` for common.